client/app/_components/ui/footer/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+import Link from 'next/link'
+import { RiCalendarFill } from 'react-icons/ri'
+import { RiHomeFill } from 'react-icons/ri'
+import { RiMoneyEuroCircleFill } from 'react-icons/ri'
+
+// TODO: This design may have some room for improvement
+const Footer = () => {
+  return (
+    // Footer: Visible on devices less wider than "md" breakpoint
+    <div className='fixed bottom-0 left-0 right-0 flex h-16 w-screen items-center bg-teal-50 text-white shadow-lg md:hidden'>
+      <div className='flex w-full items-center justify-between px-6'>
+        <Link href={'/'}>
+          <div className='flex h-10 w-10 items-center justify-center rounded-full bg-primary p-2 text-teal-50 shadow-md transition-transform duration-300 hover:scale-110'>
+            <RiHomeFill className='h-6 w-6 ' />
+          </div>
+        </Link>
+        <Link href={'/calendar'}>
```

Comment:
The center button has background color and shadow but the others do not have bg color in Figma? I think the design in Figma is better!

Reply:
I changed the icon and the design which is more similar to the figma!
https://github.com/HAK2024/fairy-share/pull/55/commits/053df127f75b384bc14ed24131a741da2409b8c0